django_geosource/tests/test_api.py

```diff
@@ -40,6 +47,36 @@ def test_list_view(self):
         self.assertEqual(response.status_code, HTTP_200_OK)
         self.assertEqual(Source.objects.count(), len(response.json()))
 
+    def test_refresh_view_fail(self):
+        source = GeoJSONSource.objects.create(
+            name="test",
+            geom_type=GeometryTypes.Point.value,
+            file=os.path.join(os.path.dirname(__file__), "data", "test.geojson"),
+        )
+        with patch(
+            "django_geosource.mixins.CeleryCallMethodsMixin.run_async_method",
+            return_value=False,
+        ):
+            response = self.client.get(
+                reverse("geosource:geosource-refresh", args=[source.pk])
+            )
+        self.assertEqual(response.status_code, HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
Should not append.

Reply:
it's at the end of the views https://github.com/Terralego/django-geosource/blob/master/django_geosource/views.py